examples/digits_dann_lightn/configs/TUTORIAL.yaml

```diff
@@ -0,0 +1,14 @@
+DAN:
```

Comment:
filename TUTORIAL, no need to be all uppercase. MN2UP-DANN etc are because they are abbreviations/acronyms of longer words.

Reply:
Thanks, changes made.